portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,88 @@
 #content {
   margin-left: auto;
   margin-right: auto;
+  margin-top: 20px;
   width: 650px;
+  padding: 20px;
+  border: 5px solid #333;
+  border-radius: 6px;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+.body{
```

Comment:
It's easier to visually scan CSS when there is a space before the curly brace -- 
```
.body {
...
}
```

Reply:
I will be sure to remember that, thanks!